builtins/amp-ad.md

```diff
@@ -64,13 +64,23 @@ Optional attribute to pass configuration to the ad as an arbitrarily complex JSO
 
```

Comment:
Add a sentence:

The `amp-ad` tag will be collapsed (set to `display: none`) if the ad sends a message that the ad slot cannot be filled and AMP determines that this operation can be performed without affecting the user's scroll position.